industrial_ci/src/tests/source_tests.sh

```diff
@@ -23,6 +23,8 @@
 # (As of version 0.4.4 most of them are defined in env.sh).
 
 function install_catkin_lint {
+    ici_install_pkgs_for_command apt-add-repository software-properties-common
+    ici_asroot apt-add-repository ppa:roehling/ros -y -u # -u for versions pre-Ubuntu 18.04
     ici_install_pkgs_for_command catkin_lint "${PYTHON_VERSION_NAME}-catkin-lint"
```

Comment:
From Ubuntu 20.04 on the package was renamed `catkin-lint`.

Reply:
If it makes things easier, there's a transitional `python3-catkin-lint` package that pulls `catkin-lint`. The package is guaranteed to be available in the next Debian stable release Bullseye (to provide an upgrade path from Buster), so all Ubuntu releases until at least Ubuntu 21.04 will still have that package, too (assuming the next Debian release happens mid-2021).

Reply:
I think for now we will use thy pypi version, because it should work for all OS (Debian/Ubuntu, back to Ubuntu trusty).

I am not 100% familiar with the Debian update policy, but if I recall it correctly only bug fixes are allowed for stable distros.
So, the user would - again - not get the latest version of `catkin-lint` after some time, correct?

Reply:
That is true. PyPI is the best choice if you're looking for the latest version in an OS independent manner. My comment was mostly aiming at my PPA, which admittedly only works for Ubuntu releases.